packages/expo-cellular/src/Cellular.ts

```diff
@@ -123,3 +186,24 @@ export async function getCellularGenerationAsync(): Promise<CellularGeneration>
   }
   return await ExpoCellular.getCellularGenerationAsync();
 }
+
+/**
+ * Gets information about the device's current cellular connection. The return value of this method
+ * changes if the device changes carriers, such as when swapping the SIM card or roaming.
+ *
+ * On Android, this method needs permission granted from
+ * [`requestPhoneStatePermissionsAsync`](#requestPhoneStatePermissionsAsync) to read information
```

Comment:
Currently, internal links are always lowercased and using uppercase character breaks them. It might something worth investigating.
```suggestion
 * [`requestPhoneStatePermissionsAsync`](#requestphonestatepermissionsasync) to read information
```